### PR TITLE
rewrite fastio

### PR DESCRIPTION
**Pull request by bqi343**
Replaced both example problems with a simpler one and timed them on the USACO servers. 

Caveat: it's not possible to submit to this.

_If any of the below doesn't apply to this Pull Request, mark the checkbox as done._

- [ ] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which includes the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making them.

### CODE REVIEW

**Comment by SansPapyrus683** [comment]
ben is it ok if i just format your code & directly commit it

**Comment by bqi343** [comment]
k

**Review by SansPapyrus683 — APPROVED** [review]
ye sure

File: content/1_General/Fast_IO.mdx [hunk 1]
```diff
@@ -64,7 +64,8 @@ less than $10^9+7$.
 Randomly generating test data results in input and output files each ~10MB
 large. It is possible to see input files this large (the 11th input file for
 [Robotic Cow Herd](http://www.usaco.org/index.php?page=viewproblem2&cpid=674) is
-~10.3MB large), though not output files (the largest I recall is due to
+~10.3MB large), though not output files (an extremely larger one is one for a
```
Comment (bqi343) [inline content/1_General/Fast_IO.mdx:67]: @SansPapyrus683 the current wording doesn't make sense

Comment (SansPapyrus683) [reply content/1_General/Fast_IO.mdx:67]: large instead of larger?

Comment (bqi343) [reply content/1_General/Fast_IO.mdx:67]: I prefer my original wording

Comment (SansPapyrus683) [reply content/1_General/Fast_IO.mdx]: first person seems kinda informal?

**Review by nwatx — APPROVED** [review]

**Comment by nwatx** [comment]
what is 'magic lines'

**Comment by bqi343** [comment]
```cpp
ios::sync_with_stdio(false);
cin.tie(nullptr);
```

**Comment by SansPapyrus683** [comment]
"unsyncing lines"

**Pull request merged**